src/lib/ListItem.js

```diff
@@ -1,14 +1,18 @@
-import React, { Component, PropTypes } from 'react'
+import React from 'react'
 
-export default class ListItem extends Component {
-  static propTypes = {
-    item: PropTypes.string.isRequired
+export default class ListItem extends React.Component {
+  static get propTypes() {
+    return {
+      children: React.PropTypes.object,
```

Comment:
Unlimited freedom of what you can do with the `react-expandable-listview`.

I can add my own custom elements as items and let `react-expandable-listview` manage their state (whether they're being open and etc).

In a current project I use `react-expandable-listview` combined with `react-router` `Link` elements to make an admin menu that's expandable.

E.g

```JavaScript
export default class Menu extends React.Component {
  static get menuItems() {
    return [
      {
        headerName: 'Products',
        isOpened: false,
        height: 100,
        items: [
          (
          <Link
            to="admin/products/all"
            className="btn btn-default"
            activeClassName="active"
          >
            All
          </Link>
          ),
          (
          <Link
            to="admin/products/expired"
            className="btn btn-default"
            activeClassName="active"
          >
            Expired
          </Link>
          ),
          (
          <Link
            to="admin/products/submitted"
            className="btn btn-default"
            activeClassName="active"
          >
            Submitted
          </Link>
          ),
        ],
      },
      {
        headerName: 'Promotions',
        isOpened: false,
        height: 100,
        items: [
          (
          <Link
            to="admin/promotions/active"
            className="btn btn-default"
            activeClassName="active"
          >
            Active
          </Link>
          ),
       ],
      },
      {
        headerName: 'Settings',
        isOpened: false,
        height: 100,
        items: [
          (
          <Link
            to="admin/settings/all"
            className="btn btn-default"
            activeClassName="active"
          >
            Al
          </Link>
          ),
        ],
      },
    ];
  }

  render() {
    return (
      <div id="admin-menu">
        <ReactExpandableListView
          data={this.constructor.menuItems}
          headerAttName="headerName"
          itemsAttName="items"
        />
      </div>
    );
  }
}
```